Rule 305

AnonymousVeNomicPlayer opened the pull request:
I thought it would be nice to introduce a board where players can move pieces and (later in the game) interact with each other, collect items or take other actions. At the moment I do only introduce a basic framework, people will have to specify actions, introduce items etc. later in the game.

Any comments on improving the initial set-up are very welcome :)

malte-l commented:
After a first quick reading, one thing I think should be changed slightly is to make sure that the message to the proposer in that turn containing the move must make clear that this is intended to be the move for that turn. I also think it should be made more explicit, that only *one* such move is allowed per turn (I am assuming this is the idea), and that eg. only the first movement command received is valid. I personally would also prefer if this was solved by requiring the email to be a separate one from the vote, which states in its subject line that it contains the movement command for the turn, and only the first such email is valid.

Snark1994 commented on board.md:
```diff
@@ -0,0 +1,5 @@
+(9,9): Running pidgeon
+(6,6): Black hole
+(7,2): Sword of a thousand truths
+(1,8): Treasure of the Illuminati
```
Alternatively, we all start on different squares, and instead of the other things, there's a Minotaur somewhere? We could then introduce rules for adding (and perhaps removing walls) and for how the Minotaur moves (e.g. there's a vote of all players on it every round until the Minotaur catches someone, and thereafter it's controlled by a vote of all players who have been caught by it; last player to be caught wins)

Snark1994 replied:
(Or it could be moved by the proposer instead of their king...)

cleborys commented:
I definitely agree that all board-related business should for now be different from the voting mechanism.

And while I find Josh's proposal intriguing, I wouldn't want a new win condition in a system I'm not yet used to.
So maybe a minor and non-repeatable reward in points instead?
Or being caught causes a loss of points? That way, at least there is an incentive to catch peoplerather than running away from the last few to be caught...

Snark1994 commented:
I would be happy the move being specified in the same email as the vote, it seems unnecessary to force people to send two. I would rather that it be the *last* email received that determines the move rather than the first, in case people change their minds.

To be clear, I was not suggesting that any of the ideas related to the Minotaur were implemented by Philipp, I generally think it's safer to have new mechanisms introduced by a range of people. But that was my suggestion for a future game direction, and if people like it then I would be happy with Philipp adding a Minotaur (that does nothing) as a placeholder.

cleborys commented:
I of course see how a single mail has a quality-of-life benefit, but I'm once again afraid to mess with the voting system.

Snark1994 commented:
In any case, this seems a fun suggestion!

AnonymousVeNomicPlayer commented:
- I have added a sentence on the case of several emails, but was actually hoping that we do not need to specify these cases each time.
- I introduced the Minotaur instead of the treasure.
- In the first draft I tried to avoid spamming the proposer of a rule with a lot of mails on different subjects. I do still think that there should be no problem with doing both a move and a vote in one email, since they can hardly be mistaken. But since more than one person seemed to be happy about getting spammed, I changed to seperate emails.

AnonymousVeNomicPlayer commented:
Maybe I should mention that I had a few more ideas on what can be implemented in the board in the next rounds. However, I did not want to introduce too many changes, so left it to other players to implement these.
- Kings could be allowed to move other players around the board.
- Players could use points to move further.
- The running pigeon could be moved by all players and people can bet where it will go next.
- Items can be dropped on the board and be picked up from there.
- Kings could fight each other or monsters appearing on the board.

cleborys commented:
As I currently read the rule, there is no moving around, if someone decides to skip his turn (as he is no "proposer of a rule")?

AnonymousVeNomicPlayer commented:
> I can see this being difficult to track in future. Can we change to a board of the style:
> ```
> ........
> B......0
> .C......
> ```
> (with us all being identified by the first letter of our surname except Clemens, who's C, for example; and numbers for other things)

I actually thought about that before and was not sure whether it is a good idea or not. For the moment the board state is easier to track with the board style format, but as soon as there are a lot of special things on the board, in particular if there are many things on a single square, the list might be advantageous. If people prefer the board, I would be happy to change.

AnonymousVeNomicPlayer commented:
@cleborys: Yes. Would you like that to be changed?

cleborys commented:
Both are fine.

Snark1994 commented:
Re: the board, I would rather for it not be possible to have multiple things on the same square, to avoid this problem. An alternative is a hybrid format, e.g.
```
.A.
..B

A = Clemens, a pigeon
B = the Minotaur
```

AnonymousVeNomicPlayer commented:
I did not want to break the symmetry in the beginning (if people start on different squares), so that no one has an advantage or disadvantage. And I guess that a hexagonal board would be much more confusing ;)
But I will introduce the hybrid format.

AnonymousVeNomicPlayer commented:
The voting was as follows:
yes: Niels B, Josh H, Peter J, Malte L, Philipp S
no: Clemens B

cleborys commented:
My "No" vote came from sending a "Yes" vote with the wrong mail account (same app, different accounts)
If that should ever happen when I vote on your rules, feel free to text me about it.
I believe that's what they call "whipping in the votes".
